electron-app/src/store/balances/getters.ts

```diff
@@ -112,6 +114,33 @@ export const getters: GetterTree<BalanceState, RotkehlchenState> = {
     }, Zero);
   },
 
+  manualBalances: (state: BalanceState) => {
+    return state.manualBalances;
+  },
+
+  manualBalanceByLocation: (state: BalanceState) => {
+    const simplifyManualBalances = state.manualBalances.map(obj => {
+      const newObj: ManualBalancesByLocation = { location: '', usdValue: Zero };
+      newObj.location = obj.location;
+      newObj.usdValue = obj.usdValue;
+      return newObj;
+    });
+    const aggregateManualBalancesByLocation: ManualBalanceByLocation = simplifyManualBalances.reduce(
+      function (
```

Comment:
nitpick: couldn't this become an arrow function?

Reply:
Done